runelite-client/src/main/java/net/runelite/client/plugins/timetracking/clocks/ClockTabPanel.java

```diff
@@ -0,0 +1,235 @@
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.timetracking.clocks;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.ActionListener;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.imageio.ImageIO;
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+import net.runelite.client.plugins.timetracking.TabContentPanel;
+import net.runelite.client.plugins.timetracking.TimeTrackingPlugin;
+import net.runelite.client.ui.ColorScheme;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.components.IconButton;
+import net.runelite.client.ui.components.shadowlabel.JShadowedLabel;
+import net.runelite.client.util.SwingUtil;
+
+public class ClockTabPanel extends TabContentPanel
+{
+	static final ImageIcon DELETE_ICON;
+	static final ImageIcon DELETE_ICON_HOVER;
+	static final ImageIcon LAP_ICON;
+	static final ImageIcon LAP_ICON_HOVER;
+	static final ImageIcon PAUSE_ICON;
+	static final ImageIcon PAUSE_ICON_HOVER;
+	static final ImageIcon RESET_ICON;
+	static final ImageIcon RESET_ICON_HOVER;
+	static final ImageIcon START_ICON;
+	static final ImageIcon START_ICON_HOVER;
+
+	private static final ImageIcon ADD_ICON;
+	private static final ImageIcon ADD_ICON_HOVER;
+
+	private final ClockManager clockManager;
+
+	private final List<ClockPanel> clockPanels = new ArrayList<>();
+
+	static
+	{
+		try
+		{
+			BufferedImage deleteIcon;
+			BufferedImage lapIcon;
+			BufferedImage pauseIcon;
+			BufferedImage resetIcon;
+			BufferedImage startIcon;
+
+			synchronized (ImageIO.class)
+			{
+				deleteIcon = ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("delete_icon.png"));
+				lapIcon = ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("lap_icon.png"));
+				pauseIcon = ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("pause_icon.png"));
+				resetIcon = ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("reset_icon.png"));
+				startIcon = ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("start_icon.png"));
+				ADD_ICON = new ImageIcon(ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("add_icon.png")));
+				ADD_ICON_HOVER = new ImageIcon(ImageIO.read(TimeTrackingPlugin.class.getResourceAsStream("add_hover_icon.png")));
```

Comment:
What's the reason for two separate images here instead of using `SwingUtil::grayscaleOffset` like the other hovered icon images?